thoth/common/openshift.py

```diff
@@ -833,6 +833,8 @@ def schedule_inspection(
         *,
         raw_specification: Optional[Dict[str, Any]] = None,
         job_id: Optional[str] = None,
+        send_messages: Optional[int] = None,
+        force_sync: Optional[int] = None,
```

Comment:
Can these become boolean? It sounds to me like they have just binary values.

Reply:
@fridex I removed these two from args to the function. Within `specification` they are stored as bools and then cast `str(int(foo))` to the template params so that they end up as `1` or `0`